ample/util/reference_manager.py

```diff
@@ -0,0 +1,265 @@
+"""Various miscellaneous functions"""
+
+__author__ = "Jens Thomas, and Felix Simkovic"
+__date__ = "16 Jul 2018"
+
+from collections import OrderedDict
+import copy
+from enum import Enum
+import os
+
+from ample.util.ample_util import is_file
+from ample.constants import SHARE_DIR
+from ample.ensembler.constants import SPICKER_RMSD, SPICKER_TM
+
+ 
+class ReferenceManager():
```

Comment:
Maybe explicitly inherit from `object`, but big big big :+1: for this refactoring.